examples/tomo/checks/check_axes_cone2d_bp.py

```diff
@@ -0,0 +1,64 @@
+"""Cone beam 2D example for checking that orientations are handled correctly.
```

Comment:
The helper isn't very useful here unfortunately since I want the angles `0, 90, 180, 270` degrees. But the helper yields `45, 135, 225, 315` since we use `uniform_partition` :-)